docs/how-tos/build-a-plugin.md

```diff
@@ -0,0 +1,192 @@
+# Build your own pynxtools plugin
+
+Your current data is not supported yet by the [built-in pynxtools readers](https://github.com/FAIRmat-NFDI/pynxtools/tree/master/src/pynxtools/dataconverter/readers) or any of the officially supported [pynxtools plugins](../reference/plugins.md)? 
+
+Don't worry, the following how-to will guide you how to write a reader your own data.
+
+
+## Getting started
+You should start by creating a clean repository that implements the following structure (for a plugin called ```pynxtools-plugin```):
+```
+pynxtools-plugin
+├── .github/workflows
+├── docs
+│   ├── explanation
+│   ├── how-tos
+│   ├── reference
+│   ├── tutorial
+├── src
+│   ├── pynxtools_plugin
```

Comment:
maybe one should make sure that "plugin" serves as a placeholder e.g. for xps
for "myownreader"

Reply:
It doesn't have to be like that. For example, `pynxtools-mpes` has two readers, `mpes` and `igor-mpes`. In principle, one plugin can have many readers.

Reply:
We tried this already using cookiecutter: https://github.com/FAIRmat-NFDI/pynxtools-plugin-template (repo is private atm). However, we didn't maintain it over time (because there was no resources for it), so it quickly became outdated. It is on my ToDo list and we may even use [cruft](https://cruft.github.io/cruft/) to automatically update all the plugins once there is a change in the template. But since it is not working, I have not included it here.